app/models/studying/primary_maths_english_grade4.rb

```diff
@@ -0,0 +1,11 @@
+module Studying
+  class PrimaryMathsEnglishGrade4 < PrimaryMathsEnglishGrade4
+    def next_step
+      if has_required_subjects == "yes" 
```

Comment:
It looks like we use true/false in some of these and yes/no in others - can they be made consistent?

Reply:
I am converting all to true/false, as I encounter them. Have done this one.